test/unit/edition_test.rb

```diff
@@ -23,23 +23,18 @@ class EditionTest < ActiveSupport::TestCase
     assert_match "Internal links must start with a forward slash", exception.message
   end
 
-  context "#auth_bypass_id" do
-    should "return a deterministic hex id if edition is in fact-check state" do
-      edition = FactoryBot.create(:edition, state: "fact_check", id: 123)
-      edition.artefact.update(kind: "help_page")
-      assert_equal edition.temp_auth_bypass_id, "a665a459-2042-4f9d-817e-4867efdc4fb8"
-    end
+  def assert_is_non_empty_string(string)
+    assert string.is_a?(String)
+    assert_not string.empty?
+  end
 
-    should "return a deterministic hex id if edition is in fact-check-received state" do
-      edition = FactoryBot.create(:edition, state: "fact_check_received", id: 123)
-      edition.artefact.update(kind: "help_page")
-      assert_equal edition.temp_auth_bypass_id, "a665a459-2042-4f9d-817e-4867efdc4fb8"
-    end
+  should "generate a unique random auth_bypass_id" do
+    edition = FactoryBot.create(:edition)
+    assert_is_non_empty_string edition.auth_bypass_id
 
-    should "return a deterministic hex id if edition is in ready state" do
-      edition = FactoryBot.create(:edition, state: "ready", id: 123)
-      edition.artefact.update(kind: "help_page")
-      assert_equal edition.temp_auth_bypass_id, "a665a459-2042-4f9d-817e-4867efdc4fb8"
-    end
+    second_edition = FactoryBot.create(:edition)
+    assert_is_non_empty_string second_edition.auth_bypass_id
+
+    assert_not_equal edition.auth_bypass_id, second_edition.auth_bypass_id
```

Comment:
I don't think you really need the previous assert_is_non_empty_string assertions here. By these not being equal you've asserted that at least one is not empty. 

I imagine some other parts of this are covered implicitly by other tests too (I assume schema would raise if they weren't a uuid for example).